proto/message.proto

```diff
@@ -1,52 +1,52 @@
 syntax = "proto3";
 
 message MessageProto {
-  oneof payload {
-    BroadcastProto broadcast = 1;
-    AgreementProto agreement = 2;
-  }
+    oneof payload {
+        BroadcastProto broadcast = 1;
+        AgreementProto agreement = 2;
+    }
```

Comment:
Sorry about my remark! You were right, of course, and we should leave the indentation at 2 spaces instead of 4 here, as is the standard for proto files.